CHANGELOG.rst

```diff
@@ -20,6 +20,8 @@ This document describes changes between each past release.
 - Server now returns 415 error response if client cannot accept JSON response (#461, mozilla-services/cliquet#667)
 - Server now returns 415 error response if client does not send JSON request (#461, mozilla-services/cliquet#667)
 - Add the ``__lbheartbeat__`` endpoint, for load balancer membership test.
+- Added the ``flush_endpoint``, ``schema`` and ``default_bucket`` capabilities
```

Comment:
But it hasn't! The protocol isn't about what's inside the resources, but how to access them, and the mean of access is exactly the same as before.